version.gradle.kts

```diff
@@ -32,4 +32,4 @@
  *
  * For dependencies on Spine SDK module please see [io.spine.dependency.local.Spine].
  */
-val protoDataVersion: String by extra("0.65.1")
+val protoDataVersion: String by extra("0.65.2")
```

Comment:
Given we introduced some breaking changes (as per the PR description), I would say it should be `0.66.0`.